mods/ts/rules/world.yaml

```diff
@@ -9,15 +9,9 @@ World:
 	MenuPaletteEffect:
 	BuildingInfluence:
 	ProductionQueueFromSelection:
-		ProductionTabsWidget: PRODUCTION_TABS
-	PaletteFromFile@playersno:
+		ProductionPaletteWidget: PRODUCTION_PALETTE
+	PaletteFromFile@player:
 		Name: player
-		Tileset: SNOW
-		Filename: unitsno.pal
-		ShadowIndex: 1
-	PaletteFromFile@playertem:
-		Name: player
-		Tileset: TEMPERAT
```

Comment:
Next time when you have a rebase problem, try to choose the variant where you have the least changes. Otherwise you are probably accidentally reverting someone else's work.